src/otx/core/model/detection_3d.py

```diff
@@ -0,0 +1,297 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+"""Class definition for 3d object detection model entity used in OTX."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import numpy as np
+import torch
+from torchvision.ops import box_convert
+
+from otx.algo.utils.mmengine_utils import load_checkpoint
+from otx.core.data.dataset.utils.kitti_utils import class2angle
+from otx.core.data.entity.base import ImageInfo
+from otx.core.data.entity.object_detection_3d import Det3DBatchDataEntity, Det3DBatchPredEntity
+from otx.core.metrics import MetricInput
+from otx.core.metrics.average_precision_3d import KittiMetric
+from otx.core.model.base import DefaultOptimizerCallable, DefaultSchedulerCallable, OTXModel
+from otx.core.types.export import TaskLevelExportParameters
+
+if TYPE_CHECKING:
+    from lightning.pytorch.cli import LRSchedulerCallable, OptimizerCallable
+    from torch import nn
+
+    from otx.core.metrics import MetricCallable
+    from otx.core.schedulers import LRSchedulerListCallable
+    from otx.core.types.label import LabelInfoTypes
+
+
+class OTX3DDetectionModel(OTXModel[Det3DBatchDataEntity, Det3DBatchPredEntity]):
```

Comment:
```suggestion
class 3DDetectionModel(OTXModel[Det3DBatchDataEntity, Det3DBatchPredEntity]):
```
Could you remove OTX?

Reply:
Actually, we use OTX prefix for all OTXModels in core/model/ folder